Shopify/cli
# Replace `deploymentGenerateSignedUploadUrl` with `appVersionGenerateSignedUploadUrl` and stop generating uuid during `uploadExtensionsBundle`

### WHY are these changes introduced?

Fixes https://github.com/Shopify/develop-app-foundations/issues/63 
Depends on https://github.com/Shopify/partners/pull/50953 and https://github.com/Shopify/partners/pull/50833


We now generate a uuid in the backend during the deploy process for both the signed upload url and the app version record itself, so we don't need to generate the uuid in the CLI for `appVersionGenerateSignedUploadUrl` or `appDeploy`.

### WHAT is this pull request doing?



### How to test your changes?


1. Create an app
2. Generate an extension
3. Run `app deploy --verbose` and verify that both `appVersionGenerateSignedUploadUrl` and `appDeploy` work without providing a uuid argument

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
